products/raspberrypi.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Raspberry Pi
+category: device
+iconSlug: raspberrypi
+permalink: /raspberry-pi
+alternate_urls:
+-   /raspberypi
+-   /raspi
+-   /rpi
+releasePolicyLink: https://wikipedia.org/wiki/List_of_iOS_and_iPadOS_devices#In_production_and_supported
+discontinuedColumn: true
+activeSupportColumn: false
+eolColumn: false
+releaseColumn: false
+releaseDateColumn: false
+
+releases:
+-   releaseCycle: "CM4"
+    releaseLabel: "Compute Module 4, Compute Module 4 Lite"
+    discontinued: 2028-01-01
+
+-   releaseCycle: "CM3-plus-lite"
+    releaseLabel: "Compute Module 3+ Lite"
+    discontinued: 2028-01-01
+
+-   releaseCycle: "CM3"
+    releaseLabel: Compute Module 3, 3 Lite, 3+
+    link: https://www.raspberrypi.com/documentation/computers/compute-module.html
+    discontinued: 2026-01-01
+
+-   releaseCycle: "pico"
+    releaseLabel: Pico
+    discontinued: 2028-01-01
+
+-   releaseCycle: "4"
+    discontinued: 2026-01-01
+
+-   releaseCycle: "3"
+    discontinued: 2026-01-01
+
+-   releaseCycle: "zero"
+    releaseLabel: "Zero 1, Zero W"
+    discontinued: 2026-01-01
+
+-   releaseCycle: "zero-2"
+    releaseLabel: "Zero 2 W"
+    discontinued: 2028-01-01
+
+-   releaseCycle: "2"
+    discontinued: 2026-01-01
+
+-   releaseCycle: "1"
+    discontinued: 2026-01-01
+
+---
+
+> [Raspberry Pi](https://www.raspberrypi.com/) is a series of small single-board computers (SBCs)
+> developed by the Raspberry Pi Foundation. It is widely used because of its low cost, modularity,
+> and open design.
+
+Some of Raspberry Pi hardware comes with an Obsolescence statement guaranteeing production till a
+specific date.
```

Comment:
@captn3m0, what do you think of adding this to the product description :

> [Raspberry Pi OS](https://www.raspberrypi.com/software/operating-systems/) always supports all Raspberry Pi models.

Reply:
I'm unsure about the second bit. The broadcom chips are powered by a closed firmware, for eg, and that will likely not remain forever supported by Broadcom?

Added a line about the OS.

Reply:
Yes, now that I think of it I am unsure about the second part too.